code/modules/antagonists/changeling/abilities/absorb.dm

```diff
@@ -149,11 +149,17 @@
 
 		var/mob/ownerMob = owner
 		target.vamp_beingbitten = 1
-		ownerMob.show_message("<span class='notice'>We must hold still...</span>", 1)
 
 		if (isliving(target))
 			target:was_harmed(owner, special = "ling")
 
+		var/datum/abilityHolder/changeling/C = devour.holder
+		if (istype(C))
+			var/datum/bioHolder/originalBHolder = new/datum/bioHolder(target)
+			originalBHolder.CopyOther(target.bioHolder)
+			C.absorbed_dna[target.real_name] = originalBHolder
```

Comment:
do we need to remove the stuff that used to happen

Reply:
wdym, like the thing that happens at the end of the action? If so, imo no, it won't cause any real issues as it just safely overwrites and addDna is called from other sources too where it is necessary to add the person to absorbed_dna.